merlin/util/spatialfeature.py

```diff
@@ -635,6 +636,28 @@ def simple_clean_cells(cells: List) -> List:
             if len(cell.get_bounding_box()) == 4 and cell.get_volume() > 0]
 
 
+def get_fov_boxes(fovs: List, alignTask: analysistask.AnalysisTask) -> List:
+    """
+    Creates a list of shapely boxes for each fov containing the global
+    coordinates as the box coordinates.
+
+    Args:
+        fovs: A list of fovs to create boxes for
+        alignTask: a globalalign task to get global coordinates for each fov
+
+    Returns:
+        A list of shapely boxes
+    """
+    coords = [alignTask.fov_global_extent(f) for f in fovs]
+    coordsDF = pandas.DataFrame(coords,
+                                columns=['minx', 'miny', 'maxx', 'maxy'],
+                                index=fovs)
```

Comment:
Seems unnecessary to go through a pandas DataFrame, but it does add a little clarity.

Reply:
Yeah that was a holdover from when I used to pass along a dataframe with the shapely boxes, I've gotten rid of it now since I just use the boxes at this point